### ISSUE TITLE
How to set the launcher or something that can remain original used import statements?

### ISSUE THREAD

**Issue by LuckyboyYu** [INRIA/spoon#3296]
It's happy to see the redundant imports in Java files are removed after processing, 
but I'm confused that all the useful imports are inserted into the class body like below.
How can I remain the classical imports format?
`import spoon.reflect.code.CtComment;.......CtComment comment = new CtJavaDocImpl()();`
------>
`spoon.reflect.code.CtComment comment = new code.demo.CtJavaDocImpl();`
Here the comment is a FieldType belongs to a class.
Thanks a lot!

**Comment by MartinWitt** [comment]
[doc](http://spoon.gforge.inria.fr/mvnsites/spoon-core/apidocs/spoon/compiler/Environment.html#setAutoImports-boolean-) should do want you want.
`launcher.getEnvironment()` for gettting the evironment object

**Comment by LuckyboyYu** [comment]
Oh thanks!
I initiate the launcher by doing this:
`init(ConstantUtil.PROCESSOR_ROOT_PATH);   launcher.setArgs(args);`
init()is a method with static and public modifiers，I set some environment properties in it, including setAutoImports()， it just doesn't work at all.
I reversed the order of these two statements just now and it worked, I guess the setArgs() method may reset some options of launcher.
Thank you MartinWitt!
